de/manual/ch_Tools_Assistants.xml

```diff
@@ -70,43 +70,39 @@
-        <footnote id="fn_tools_all_accounts">
-          <para>Die Auswahl <guilabel>Alle Konten</guilabel> führt eine Suche durch, bei der Buchungen in den
-            Konten, die im Dialog <guilabel>Konten wählen</guilabel> markiert wurden, gefunden
-            werden, die mindestens einen Buchungsteil in jedem der ausgewählten Konten enthalten.
-            Wenn die Suche in einem regulären Kontobuch gestartet wird, ist dies das einzige
-            Kriterium, das Buchungsteile von anderen Konten prüft. Wenn Sie also den Dialog
-            <guilabel>Buchungen suchen</guilabel> z.B. in Aktiva:Umlaufvermögen:Girokonto öffnen
-            und alle Buchungen dieses Kontos mit einem Buchungsteil in Ausgaben:Lebensmittel finden
-            wollen, ist dies als Kriterium zu verwenden. Beachten Sie jedoch, dass es sich um eine
-            <emphasis>UND</emphasis>-Suche handelt: Wenn Sie zusätzlich auch nach
-            Ausgaben:Verschiedenes suchen, erhalten Sie <emphasis>nur</emphasis> die Buchungen mit
-            einem Buchungsteil in jedem der beiden Konten und <emphasis>nicht</emphasis> die
-            Buchungen mit nur einem einzigen Buchungsteil.
-          </para>
-        </footnote>
-        Möchten Sie stattdessen das gesamte &app;-Buch durchsuchen, dann starten Sie die Suche
-        entweder in der <emphasis>Kontenübersicht</emphasis>, oder Sie öffnen das
-        &mc.to.gen-jour;.
-      </para>
+      <para>Abhängig davon, wie tief Sie in Ihren Daten abtauchen wollen, werden die folgenden
+        <emphasis>Ausgangssituationen</emphasis> unterschieden:
+        <variablelist>
+          <varlistentry>
+            <term>Ein einzelnes <emphasis>Konto</emphasis> durchsuchen</term>
 
-      <para>Bei den Buchungsfeldern werden nur die Buchungen berücksichtigt, die mit einem Buchungsteil in
-        diesem Register verknüpft sind, während bei den übrigen Buchungen nur die mit dem Konto
-        verbundenen Buchungsteile durchsucht werden, die diesem Konto zugeordnet sind, außer Sie
-        verwenden die Suchfunktion <guilabel>Alle Konten</guilabel>.
+            <listitem>
+              <para>Führen Sie die Suche in der Registerkarte eines Kontos durch, so werden nur die zu diesem Konto
+                zugehörigen Buchungsteile durchsucht.
+              </para>
+            </listitem>
+          </varlistentry>
+
+          <varlistentry>
+            <term>Das gesamte <emphasis>&appname;-Buch</emphasis> durchsuchen</term>
+
+            <listitem>
+              <para>Starten Sie die Suche entweder in der <emphasis>Kontenübersicht</emphasis>, oder in dem geöffneten
+                <xref linkend="general-journal" />, so wird die Suche über alle Konten ausgeführt.
+              </para>
+            </listitem>
+          </varlistentry>
+        </variablelist>
       </para>
 
       <para>Wenn Sie zuvor die Buchungen in einem Register mit &mc.vw.f-by; filtern, dann werden in diesem Konto
```

Comment:
Ich binmir nicht ganz sicher, ob wir den Begriff Register (an anderer Stelle) hinreichend definiert haben. Technisch ist es in Gnucash der Typ/Oberbegriff von Ledger (Kontenblatt) und Journal für ihre einheitliche Darstellung..

Reply:
In der Tat verwenden wir den Begriff `Register` in Guide und Manual selten.